.github/workflows/commitint.yml

```diff
@@ -0,0 +1,23 @@
+name: commitlint
+
+on:
+  pull_request:
+
+jobs:
+  run-commitlint-on-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Validate PR commits with commitlint
+        if: github.event_name == 'pull_request'
+        run: npx commitlint --from ${{ github.event.pull_request.base.sha }} --to ${{ github.event.pull_request.head.sha }} --verbose
```

Comment:
funny story.... i think that commitlint action we saw should actually work for us - i found that it [accepts the commitlint.config.js](https://github.com/wagoid/commitlint-github-action/blob/master/action.yml#L5-L9), so it should use our custom commit types.

in any case, i advocate to keep your implementation until we run into a bug or issue that the repo action has solved. your implementation is much simpler. the only thing i currently see is that the repo action will output a [link to explain the convention](https://github.com/wagoid/commitlint-github-action/blob/master/action.yml#L25-L28), but i think maybe we can do this too if the messaging is unclear. i've got to see what the messaging looks like when it fails